capstone/src/main/webapp/script.js

```diff
@@ -23,17 +37,17 @@ function createMap() {
 
   //* *button for centering map*/
   const centerControlDiv = document.createElement('div');
-  const centerControl = new CenterControl(centerControlDiv, map);
+  var centerControl = new CenterControl(centerControlDiv, map);
```

Comment:
You shouldn't be using var (see google style guides). Doesn't look like this is being reassigned so const should be good.

Reply:
Actually, Jin suggested I use var, but I fixed all of the eslinter suggestions except for google. Which did not change when adding the comments you suggested.